setup.py

```diff
@@ -59,6 +59,8 @@
         'invenio-sipstore>=1.0.0a1',
     ],
     'deposit': [
+        # Temporary pin to avoid collision with Flask-Security
```

Comment:
You should not get Flask-Security when you install plain Invenio-Deposit (https://github.com/inveniosoftware/invenio-deposit/blob/master/setup.py#L65-L85). It is probably coming from Invenio-Accounts installed elsewhere.